connectivity/tests/world.go

```diff
@@ -32,10 +32,9 @@ func (s *podToWorld) Name() string {
 }
 
 func (s *podToWorld) Run(ctx context.Context, t *check.Test) {
-	chttp := check.HTTPEndpoint("cilium-io-http", "http://cilium.io")
-	chttps := check.HTTPEndpoint("cilium-io-https", "https://cilium.io")
-	chttpindex := check.HTTPEndpoint("cilium-io-http-index", "http://cilium.io/index.html")
-	jhttp := check.HTTPEndpoint("jenkins-cilium-io-http", "http://jenkins.cilium.io")
+	http := check.HTTPEndpoint("one-one-one-one-http", "http://one.one.one.one")
+	https := check.HTTPEndpoint("one-one-one-one-https", "https://one.one.one.one")
+	httpsindex := check.HTTPEndpoint("one-one-one-one-https-index", "https://one.one.one.one/index.html")
```

Comment:
I am probably missing context but is that check really needed?

Reply:
Do you mean the check over HTTPS specifically or the check for `/index.html`?

If it's the latter, I think this is needed for L7 policy testing, see commit e54d39bc43e6884dd66fd2ae8ea7c674da10fedc /cc @jrajahalme

Reply:
The check for `index.html`, yes. OK!